app/models/user.rb

```diff
@@ -9,4 +9,43 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  has_many :invitations_i_got, foreign_key: :invitee_id, class_name: 'Friendship'
+  has_many :who_invited_me, through: :invitations_i_got, source: :inviter
+
+  has_many :invitations_i_sent, foreign_key: :inviter_id, class_name: 'Friendship'
+  has_many :who_i_invited, through: :invitations_i_sent, source: :invitee
+
```

Comment:
## One-to-many relation
- Once you have updated the User--> friendship relation with a migration, com to there and update this relation.
So far this relation is many-to-many.
_Use this [hint ](https://smartfunnycool.com/friendships-in-activerecord/)_